fitwsarah_frontend/src/components/AdminPanel/Filter.js

```diff
@@ -79,13 +92,29 @@ function Filter({src, labels, onInputChange, searchTerm, clearFilters}) {
                                         )}
                                         {src === "feedback" && (
                                         <>
-                                        {/*The names are the same english or french*/}
+                                        {}
```

Comment:
[nitpick]: you should probably get rid of useless brackets.

Reply:
no worries thx for ur comment, ill do it right away.